frontend/app/controllers/Staff.scala

```diff
@@ -30,6 +31,16 @@ trait Staff extends Controller {
     Ok(views.html.eventOverview.details(request.path))
   }
 
+  // Temporary solution to evaluate SessionCam without actually activating it across the entire site
+  def dropSessionCamCookie = GoogleAuthenticatedStaffAction { implicit request =>
```

Comment:
Remove `GoogleAuthenticatedStaffAction`  - shouldn't need to authenticate to try this out!